perception/autoware_radar_object_tracker/src/tracker/model/constant_turn_rate_motion_tracker.cpp

```diff
@@ -280,12 +280,10 @@ bool ConstantTurnRateMotionTracker::predict(const double dt, KalmanFilter & ekf)
   // Rotate the covariance matrix according to the vehicle yaw
   // because q_cov_x and y are in the vehicle coordinate system.
   Eigen::MatrixXd Q_xy_local = Eigen::MatrixXd::Zero(2, 2);
-  Eigen::MatrixXd Q_xy_global = Eigen::MatrixXd::Zero(2, 2);
   Q_xy_local << ekf_params_.q_cov_x, 0.0, 0.0, ekf_params_.q_cov_y;
   Eigen::MatrixXd R = Eigen::MatrixXd::Zero(2, 2);
   R << cos(yaw), -sin(yaw), sin(yaw), cos(yaw);
-  Q_xy_global = R * Q_xy_local * R.transpose();
-  Q.block<2, 2>(IDX::X, IDX::X) = Q_xy_global;
+  Q.block<2, 2>(IDX::X, IDX::X) = R * Q_xy_local * R.transpose();
```

Comment:
Personally, I find the previous version easier to read. How about this one?



```suggestion
  Eigen::MatrixXd Q_xy_global = R * Q_xy_local * R.transpose();
  Q.block<2, 2>(IDX::X, IDX::X) = Q_xy_global;
```